### ISSUE TITLE
include plgx-Plugin file in releases

### ISSUE THREAD

**Issue by itc-ger** [kapsiR/HaveIBeenPwnedKeePassPlugin#12]
Could you create and provide an **plgx plugin file** in your releases, please?
I'd like to create an NixOS package of your plugin. And the plgx would be more suitable in this case for compatibility reasons.

**Comment by kapsiR** [comment]
I'll consider this but I can't promise that I'll do that.
Currently, I'm not producing a `plgx` because there are limitations regarding the C# version applied.

**Comment by itc-ger** [comment]
Thanks for your quick response. I can see your point. 
So I'll keep watching your repos releases ;-)

**Comment by kapsiR** [comment]
I took some brief research in current open issues of KeePass itself, if it would be worth to invest some time here, but I decided against it. I've planned to introduce some changes in the next months, where I don't want to take care of all that stuff that doesn't work with C# 5.

Sorry for that, I hope you still use the extension anyway. 
You mentioned compatibility reasons... on that topic I can say there are no plans to switch off the current .NET Framework version which I defined in the system requirements (it's almost the latest anyway)
